lib/ok.ex

```diff
@@ -150,6 +150,62 @@ defmodule OK do
   def is_failure?({:ok, _value}), do: false
   def is_failure?({:error, _reason}), do: true
 
+  @doc guard: true
```

Comment:
Is this meta data necessary even when using defguard? seems like that could handle it

Reply:
This is not necessary when using a guard, it would only generate another section in your docs, much like the Elixir docs.  The two guard would be in a "Guards" section.

Reply:
Ah so it's only necessary because you added the groupings to `mix.exs` :+1: